LayoutTests/inspector/view/asynchronous-layout-expected.txt

```diff
@@ -31,22 +31,31 @@ PASS: Parent view should have 0 dirty descendants.
 PASS: Child view should have 0 dirty descendants.
 PASS: Parent view should have started a layout.
 PASS: Child view should have completed 1 layout.
-Child view completed a layout.
-PASS: Root view should have 0 dirty descendants.
-PASS: Parent view should have 0 dirty descendants.
-PASS: Child view should have 0 dirty descendants.
-PASS: Parent view should have started a layout.
-PASS: Child view should have completed 2 layouts.
```

Comment:
I think this is because of your changes to `WI.View.prototype.updateLayout`, but looking at the test I'm slightly concerned because the test is supposed to end after "Parent view completed a layout.", so we should either update the test or maybe even rework how `WI.TestView.prototype.evaluateAfterLayout` works 🤔

Reply:
Fixed the test @patrickangle and myself talked about it today. We realized there's a missing callback for the _second_ call to `parent.didLayoutSubtree()`,  which is where the `resolve()` should actually go. 

Our changed implementation of `WI.View` will call it on the parent view for each layout of the subview:
2x subview layout  -> 2x `parent.didLayoutSubtree()`